IMU Transformation AL article & Small IMU Docs Update (#706)
pupil-labs/pupil-docs: neon/data-collection/data-streams/index.md

<Badge>Real-time</Badge><Badge>Pupil Cloud</Badge><Badge>Neon Player</Badge>
The Neon module is equipped with a 9-DoF [inertial measurement unit](https://invensense.tdk.com/products/motion-tracking/9-axis/icm-20948/) (IMU) featuring an accelerometer, gyroscope, and magnetometer. The accelerometer and gyroscope measure linear acceleration and angular velocity, respectively, and are provided as raw values.

> A fusion engine also combines these values with magnetometer readings to estimate the module's absolute orientation relative to magnetic north (positive world y-axis), gravity (negative world z-axis), and a rightward pointing vector (positive world x-axis) as a quaternion. We refer to this as the world coordinate system. It is important to note that this is not the same as the local IMU coordinate system.

Author: ```suggestion
A fusion engine also combines these values with magnetometer readings to estimate the module's absolute orientation in the world as a quaternion. The world coordinate system is defined such that magnetic north corresponds to the positive y-axis, gravity corresponds to the negative z-axis, and a rightward pointing vector to these corresponds to the positive x-axis.
```
I propose this slightly altered version. IMO a formulation that says "The world coordinate system is [...]" is easier to understand.

Reviewer: Yes, thanks!

The IMU is located in the top bar of the module and is sampled at 110 Hz. Its local coordinate system is oriented with the x-axis pointing to the right, the y-axis pointing in front, and the z-axis pointing upwards.

![IMU Coordinate System](./imu-xyz-black.jpg)

When relating data from the IMU to things visible in the scene camera, it may be necessary to align their respective 3D coordinate systems. The IMU's coordinate system is rotated by 102° around the x-axis in relation to the scene camera's coordinate system. See our [IMU Transformations article](https://docs.pupil-labs.com/alpha-lab/imu-transformations/) for a guide.

![IMU Scene Camera](./imu-scene_camera_offset-black.png)

### Euler Angles

When exporting recordings from Pupil Cloud or Neon Player the IMU's orientation in Euler angles (i.e. pitch, yaw, and roll) is also available:

- Pitch is defined as a rotation around the world x-axis with a value range of -90° to +90°. Wearing Neon upright and looking parallel to the horizon roughly corresponds to 0° pitch. Backward tilt is positive, forward is negative.

> - Yaw is a rotation around the world z-axis with a value range of -180° to +180°. With a calibrated IMU, a yaw of 0° indicates alignment with magnetic north. Leftward turn is positive, rightward is negative.

Author: ```suggestion
- Yaw is a rotation around the world z-axis with a value range of -180° to +180°. With a [calibrated IMU](/data-collection/calibrating-the-imu/), a yaw of 0° indicates alignment with magnetic north. Leftward turn is positive, rightward is negative.
```
